samples/stockquote/deployable-jar/src/main/java/org/wso2/msf4j/stockquote/example/Stock.java

```diff
@@ -0,0 +1,90 @@
+/*
```

Comment:
Added README.md for the deployable-jar sample. commit revision 7f4b898fcb181261c0d9d51914a9d80548a41843
Thanks